awx/main/dispatch/worker/callback.py

```diff
@@ -91,7 +91,7 @@ def flush(self, force=False):
                     for e in events:
                         try:
                             if (
-                                isinstance(exc, IntegrityError),
+                                isinstance(exc, IntegrityError) or
```

Comment:
@ryanpetrello Is there any particular reason for the `comma` syntax here?  I tested this out and `or` is functionally the same.  Just wanted to check as you wrote this and it is some mission critical code.

Reply:
No, this is a bug, and it should be `and`.  I probably meant to wrap this in `all()`.